blackbox/docs/sql/reference/grant.txt

```diff
@@ -0,0 +1,35 @@
+.. _ref-grant:
+
+=========
+``GRANT``
+=========
+
+Grants privilege to an existing user.
+
+Synopsis
+========
+
+.. code-block:: psql
+
+  GRANT { { READ | WRITE [,...] } | ALL [ PRIVILEGES ] }
```

Comment:
pls add a space after the comma for the `can be repeated` syntax like that: `... WRITE [, ...]`